.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       - name: Set up PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: '8.3'
```

Comment:
Please revert this. For 5.x branch our lowest supported version is 7.4, and PHPCS (Slevomat coding standard) uses the version it runs on to customize some sniffs. Thus it should match the lowest version we support.